### PR TITLE
feat(dashboards): Add errors widgets to new timeseries visualization allowlist

### PR DESCRIPTION
**Pull request by edwardgou-sentry**

### CODE REVIEW

File: static/app/views/dashboards/widgetBuilder/components/newWidgetBuilder.spec.tsx [hunk 1]
```diff
@@ -72,7 +72,14 @@ describe('NewWidgetBuilder', () => {
 
     MockApiClient.addMockResponse({
       url: '/organizations/org-slug/events-stats/',
-      body: [],
+      body: {
```
Comment (edwardgou-sentry) [inline static/app/views/dashboards/widgetBuilder/components/newWidgetBuilder.spec.tsx:75]: needed to add a mock return because this updates some tests to use the new visualization, which throws an error on empty plottables.

**Review by narsaynorath — APPROVED** [review]
